instance/models/load_balancer.py

```diff
@@ -103,20 +111,16 @@ class LoadBalancingServer(ValidateModelMixin, TimeStampedModel):
     accepts_new_backends = models.BooleanField(default=True)
     # A random postfix appended to the haproxy configuration file names to make collisions
     # impossible.
-    fragment_name_postfix = models.CharField(max_length=8, blank=True)
+    fragment_name_postfix = models.CharField(
+        max_length=8,
+        blank=True,
+        default=functools.partial(generate_fragment_name, length=8),
```

Comment:
Nit: Since `generate_fragment_name` isn't used elsewhere you could just make `length=8` the default parameter value, and then you don't need to import functools nor apply a partial.